db/migrate/20240402194159_create_version_contexts.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateVersionContexts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :version_contexts do |t|
+      t.string      :druid, null: false
+      t.integer     :version, null: false, default: 1
+      t.jsonb       :values, default: {}
```

Comment:
I generally don't like the idea of naming an attribute with something that is already part of the model name, because it then leads to code that looks like `version_context.context`.  Personally I don't like this and find it less readable than something like `version_context.values` or `version_context.data` or whatever.